railties/test/generators/app_generator_test.rb

```diff
@@ -259,6 +259,20 @@ def test_config_jdbc_database_when_no_option_given
     end
   end
 
+  def test_generator_without_skips
+    run_generator
+    assert_file "config/application.rb", /\s+require\s+["']rails\/all["']/
+    assert_file "config/environments/development.rb" do |content|
+      assert_match(/config\.action_mailer\.raise_delivery_errors = false/, content)
+    end
+    assert_file "config/environments/test.rb" do |content|
+      assert_match(/config\.action_mailer\.delivery_method = :test/, content)
+    end
+    assert_file "config/environments/production.rb" do |content|
+      assert_match(/# config\.action_mailer\.raise_delivery_errors = false/, content)
+    end
+  end
```

Comment:
It wasn't clear to me this intention under the test, because most of what it checks is action mailer related content. Anyway, thanks for your work on this :+1: